frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -63,6 +63,7 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
   const { projectName } = startNotebookData;
   const navigate = useNavigate();
   const [createInProgress, setCreateInProgress] = React.useState(false);
+
```

Comment:
Nit. Typically you want to keep your changes to a min in files you don't want to touch. This kind of stuff is usually stuff you should trim from your commit.